### PR TITLE
Fix embedded images not rendered

### PR DESCRIPTION
**Pull request by velyan**

### CODE REVIEW

File: src/backend_skia_bindings/image.rs [hunk 1]
```diff
@@ -62,7 +64,7 @@ fn image_to_surface(image: &Image, surface: &mut [u8]) {
     use rgb::FromSlice;
 
     let mut i = 0;
-    if skia::ColorType::n32() == skia::ColorType::RGBA8888 {
+    if skia::ColorType::n32() == skia::ColorType::BGRA8888 {
```
Comment (velyan) [inline src/backend_skia_bindings/image.rs:67]: same as this - https://github.com/RazrFalcon/resvg/blob/b70a71293d86e4bd058970600668e4c9217810b5/resvg-skia/skia-bindings/skia_capi.cpp#L170

File: src/backend_skia_bindings/image.rs [hunk 2]
```diff
@@ -21,12 +21,13 @@ pub fn draw_raster(
 
     let mut image = {
         let (w, h) = img.size.dimensions();
-        let surface = usvg::try_opt_warn_or!(
-            skia::Surface::new_raster(&skia::ImageInfo::new_unknown(Some(skia::ISize::new(w as i32, h as i32))), None, None), (),
+        let image_info = skia::ImageInfo::new(skia::ISize::new(w as i32, h as i32),skia::ColorType::n32(), skia::AlphaType::Unpremul, None);
```
Comment (velyan) [inline src/backend_skia_bindings/image.rs:24]: same as this - https://github.com/RazrFalcon/resvg/blob/b70a71293d86e4bd058970600668e4c9217810b5/resvg-skia/skia-bindings/skia_capi.cpp#L67

**Review by therealbnut — APPROVED** [review]
LGTM

**Review by romanzes — COMMENTED** [review]
Would rebasing on top of latest resvg be too difficult?

**Review by StewartCanva — APPROVED** [review]

**Comment by velyan** [comment]
> Would rebasing on top of latest resvg be too difficult?

I guess not, but we need to test first and see what has changed. Can do it as a separate PR. Is there something on latest resvg master that we need?

**Comment by romanzes** [comment]
> Is there something on latest resvg master that we need?

IIUC, the fix for this bug is there, and in this PR we're applying it independently, right?

**Comment by velyan** [comment]
> > Is there something on latest resvg master that we need?
> 
> IIUC, the fix for this bug is there, and in this PR we're applying it independently, right?

Oh actually, it was only an issue with our bindings, so nothing to do with resvg master. This is the fix

**Pull request merged**